org.eclipse.transformer/pom.xml

```diff
@@ -25,6 +25,9 @@
 	<description>Eclipse Transformer Library</description>
 	<name>${project.groupId}:${project.artifactId}</name>
 	<url>https://projects.eclipse.org/projects/technology.transformer</url>
+	<properties>
+		<maven.compiler.release>9</maven.compiler.release>
```

Comment:
This is needed to avoid compilation error of the following form:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.13.0:compile (default-compile) on project org.eclipse.transformer: Compilation failure
[ERROR] /Users/jluehe/Desktop/ws/transformer/org.eclipse.transformer/src/main/java/org/eclipse/transformer/action/impl/ZipActionImpl.java:[196,44] cannot find symbol
[ERROR]   symbol:   method transferTo(java.io.ByteArrayOutputStream)
[ERROR]   location: variable inputStream of type java.io.InputStream
```

Reply:
We still need to target Java 8 at this time. This is why we use Bnd 6.4 and not Bnd 7. So you cannot introduce a dependency on a later version of Java.

If we decide to drop Java 8 support, I would move to Java 17 or 21.

Reply:
@bjhargrave, thx for the explanation! Made the required change to not depend on Java 9 or greater.